### PR TITLE
Clarify that the "Build libyuv" step uses clang-cl

### PR DESCRIPTION
**Pull request by wantehchang**

### CODE REVIEW

**Comment by wantehchang** [comment]
This pull request is intended to find out if the Windows CI checks are still failing.

**Review by jzern — APPROVED** [review]

File: .github/workflows/ci-windows.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
         -DAVIF_ENABLE_WERROR=ON
     - name: Build libavif (ninja)
       working-directory: ./build
-      run: ninja
+      run: ninja -v
```
Comment (wantehchang) [inline .github/workflows/ci-windows.yml:76]: It is strange that after I added the `-v` option to the `ninja` command, the command lines of the commands executed by `ninja` are still not shown. So the only thing we see is the error message from `ninja`:

```
ninja: error: '/defaultlib:msvcrt', needed by 'avif_example_decode_memory.exe', missing and no known rule to make it
```

Comment (wantehchang) [reply .github/workflows/ci-windows.yml:76]: @vrabaud I added a `#error` directive to src/read.c to force a compilation error. Yet the Windows CI build did not seem to compile src/read.c. Something strange is wrong with the Windows build.

Comment (wantehchang) [reply .github/workflows/ci-windows.yml:76]: The ninja error message shows that ninja thinks `/defaultlib:msvcrt` is a build target that the build target `avif_example_decode_memory.exe` depends on. So the source of the bug may be that the linker option `/defaultlib:msvcrt` is incorrectly treated as a linker input file.

In the output of the "Prepare libavif (cmake)" step, I found an evidence of this mistake:

```
-- libavif(AVIF_CODEC_RAV1E=LOCAL): compiled rav1e library not found at D:/a/libavif/libavif/ext/rav1e/build.libavif/usr/lib/rav1e.lib; using FetchContent
-- libavif(AVIF_CODEC_RAV1E=LOCAL): fetching and configuring rav1e
-- Rust Toolchain: stable-x86_64-pc-windows-msvc
-- Rust Target: x86_64-pc-windows-msvc
-- Determining required link libraries for target x86_64-pc-windows-msvc
-- Required static libs for target x86_64-pc-windows-msvc: kernel32;advapi32;kernel32;ntdll;userenv;ws2_32;kernel32;ws2_32;kernel32;$<$<CONFIG:Debug>:msvcrtd>;/defaultlib:msvcrt
-- Found Rust: C:/Users/runneradmin/.rustup/toolchains/stable-x86_64-pc-windows-msvc/bin/rustc.exe (found version "1.79.0")
-- Using Corrosion as a subdirectory
-- libavif(AVIF_CODEC_RAV1E=LOCAL): fetching and configuring rav1e - complete
-- libavif: Codec enabled: rav1e (encode)
```

Comment (wantehchang) [reply .github/workflows/ci-windows.yml:76]: I tracked down the bug in cmake/Modules/LocalRav1e.cmake. I will abandon this test pull request. The fix is https://github.com/AOMediaCodec/libavif/pull/2219.

**Pull request closed without merging**